libs/wingc/src/jsify.rs

```diff
@@ -693,7 +695,90 @@ impl<'a> JSifier<'a> {
 						"\")"
 					)
 				}
-				_ => new_code!(expr_span, ""),
+				IntrinsicKind::Inflight => {
+					let arg_list = intrinsic.arg_list.as_ref().unwrap();
+					let path = if let ExprKind::Literal(Literal::String(ss)) = &arg_list.pos_args[0].kind {
+						// trim off the first and last character (the quotes)
+						let ss = &ss[1..ss.len() - 1];
+						let extern_path = Utf8Path::new(&ss);
+
+						let extern_path = normalize_path(extern_path, ctx.source_path);
+						if !extern_path.exists() {
+							todo!()
+							// self.spanned_error(exp, format!("Extern file '{}' does not exist", extern_path));
+						}
+						extern_path
+					} else {
+						return new_code!(&expression.span, "");
+					};
+
+					let mut export_name = "default";
+
+					let mut lifts: IndexMap<String, (&Expr, Option<&Vec<Expr>>)> = IndexMap::new();
+					for x in &arg_list.named_args {
+						if x.0.name == "export" {
+							if let ExprKind::Literal(Literal::String(s)) = &x.1.kind {
+								export_name = &s[1..s.len() - 1];
+							}
+						} else if x.0.name == "lifts" {
```

Comment:
There is a std type and it does go through normal type checking.
This code here is not parsing it's just traversing the AST node.

The reason it's like this is because the lifts need to be statically known (to know which types to grab)